server/src/main/java/io/crate/execution/engine/join/HashJoinBatchIterator.java

```diff
@@ -43,6 +44,8 @@
  *     Build Phase:
  *     for (leftRow in left) {
  *         calculate hash and put in Buffer (HashMap) until the blockSize is reached
+ *         // for left outer set a marker which indicates that a pair was found
+ *         buffer.marker = false
```

Comment:
The comments could also use an update for the marker -> matched change to reflect the code

Reply:
👍